src/pages/StudentDashboard/StudentDashboard.test.js

```diff
@@ -24,15 +22,12 @@ describe("<StudentDashboard />", () => {
     const heading = screen.getByRole("heading", {
       name: /weekly assessments/i
     })
-    const logo = screen.getByRole("img", {
-      name: /learn logo/i
-    })
+
```

Comment:
Same thing with the `it("renders without crashing", () => {` test in this file. You have better tests now so let's just get rid of the one that keeps getting flagged by github actions.